### PR TITLE
Modify tags to suit Track-O better

### LINKED ISSUE #114: [PE-D] Phone number regex might not be strict enough
While adding a new tutee, this command `add add n/Mary p/934 sch/Temasek Jc l/j2 a/135 Rivervale Link` is allowed even though numbers in Singapore should have 8 numbers to be valid. Even though its not a major bug but I feel that it could potentially bring some inconvenience for the user since there is no checks implemented. 

Should be a simple fix by just changing the regex used!

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: tenebrius1/ped#4

### LINKED ISSUE #120: [PE-D] Find command format in error message differs from UG specification
UG mentions only keyword `yes` or `no` is accepted for the `find` command's `overdue` filter.

However, the error message when just `find` is input in command box, instructs the user to input either `true` or `false` for `overdue` filter. Input accepted by the app is `yes` or `no`.

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: CraveToCode/ped#2

### LINKED ISSUE #146: [PE-D] Minor UI Bug
The space between the day and subject seems to be quite wide. Maybe you all can consider changing it?

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: tenebrius1/ped#13

### LINKED ISSUE #149: [PE-D] Use of tags
Not exactly a bug, but I was wondering if the use of tags is really necessary in this app.

Like in the sample tutees, the tags show relationships like "friend" or "neighbour" but in the end of the day, isn't everyone just a tutee? XD I also feel like since there is already a remark present the tag becomes less useful, but i don't think this is a bug hahaha just my opinion :")

-------------
Labels: `type.FeatureFlaw` `severity.Low`
original: feliciaivane/ped#12

### PR DESCRIPTION
**Pull request by KengXIII**
Modified some of the tags to suit the purpose of track-o better - resolves #149 
Replaced 4 spaces with a hyphen for the toString method of Lesson for better readability - resolves #146 
Updated the regex of phone numbers to only accept 8 digit numbers <b>after</b> trimming - resolves #114 
Updated the find command message from true/false to yes/no - resolves #120

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java [hunk 1]
```diff
@@ -47,8 +47,8 @@
     public static final String VALID_LEVEL_BOB = "s2";
     public static final String VALID_ADDRESS_AMY = "Block 312, Amy Street 1";
     public static final String VALID_ADDRESS_BOB = "Block 123, Bobby Street 3";
-    public static final String VALID_TAG_HUSBAND = "husband";
-    public static final String VALID_TAG_FRIEND = "friend";
+    public static final String VALID_TAG_HUSBAND = "practical";
+    public static final String VALID_TAG_FRIEND = "rescheduled";
```
Comment (botr99) [inline src/test/java/seedu/address/logic/commands/CommandTestUtil.java:51]: Perhaps we could refactor the static variable name too

**Review by botr99 — APPROVED** [review]
Resolves all the issues stated. LGTM!

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java (hunk 1 above)
Comment (KengXIII) [reply src/test/java/seedu/address/logic/commands/CommandTestUtil.java]: Agree, I updated them in the next commit!

**Review by nhjryan — APPROVED** [review]
LGTM! The tags are a lot clearer and appropriate now!

**Pull request merged**